src/group_theory/congruence.lean

```diff
@@ -846,4 +847,36 @@ def quotient_quotient_equiv_quotient (c d : con M) (h : c ≤ d) :
     show _ = d.mk' a * d.mk' b, by rw ←d.mk'.map_mul; refl,
   ..quotient_quotient_equiv_quotient c.to_setoid d.to_setoid h }
 
+end monoids
+
+section groups
+
+variables {M} [group M] [group N] [group P] (c : con M)
+
+/-- Multiplicative congruence relations preserve inversion. -/
+@[to_additive "Additive congruence relations preserve negation."]
+protected lemma inv : ∀ {w x}, c w x → c w⁻¹ x⁻¹ :=
+λ x y h, begin
+  have := c.symm (c.mul (c.mul (c.refl x⁻¹) h) (c.refl y⁻¹)),
+  rwa [mul_left_inv, one_mul, mul_inv_cancel_right] at this,
+end
+
+/-- The inversion induced on the quotient by a congruence relation on a type with a
+    inversion. -/
+@[to_additive "The negation induced on the quotient by an additive congruence relation on a type
+with an negation."]
+instance has_inv : has_inv c.quotient :=
+⟨λ x, quotient.lift_on' x (λ w, ((w⁻¹ : M) : c.quotient))
+     $ λ x y h, c.eq.2 $ c.inv h⟩
+
+/-- The quotient of a group by a congruence relation is a group. -/
+@[to_additive "The quotient of an `add_group` by an additive congruence relation is
+an `add_group`."]
```

Comment:
Let's try to be consistent with the quotes here!
```suggestion
/-- The quotient of a `group` by a congruence relation is a `group`. -/
@[to_additive "The quotient of an `add_group` by an additive congruence relation is
an `add_group`."]
```

Reply:
I guess this is wrong higher up in the file too - I'll fix both cases.

Reply:
Hmm, actually this entire file uses "monoid" as a word but `add_monoid` as code - perhaps because the latter isn't a word by itself? I'd be inclined to leave this be for consistency - maybe we should do a mathlib-wide find-and-replace for `monoid` without backticks...

Reply:
I think it is fine to keep this without the backticks.

bors d+